docs/source/user-guide/lazy/execution.md

```diff
@@ -53,27 +53,17 @@ We look at [streaming in more detail here](streaming.md).
 
 While you're writing, optimizing or checking your query on a large dataset, querying all available data may lead to a slow development process.
 
-You can instead execute the query with the `.fetch` method. The `.fetch` method takes a parameter `n_rows` and tries to 'fetch' that number of rows at the data source. The number of rows cannot be guaranteed, however, as the lazy API does not count how many rows there are at each stage of the query.
+You can instead limit the number of scanned partitions or use .head early in the query when testing. Keep in mind that aggregations and filters may behave unpredictably on subsets of data.
```

Comment:
```suggestion
Instead, you can scan a subset of your partitions or use `.head`/`.collect` at the beginning and end of your query, respectively.
Keep in mind that the results of aggregations and filters on subsets of your data may not be representative of the result you would get on the full data.
```

I'm avoiding the word “limit” because of the method with the same name, `limit`.
I'm also avoiding the term “unpredictable” because the behaviour is quite predictable; it's the user who might forget that the results may not be representative because we're only working with a subset of the data.

Do these comments make sense to you?

Reply:
Sure, makes sense.